src/tasks.md

```diff
@@ -0,0 +1,263 @@
+  ```
+
+- *Evaluate the policy*: assess the policy and, if available, find the right configuration values to match your requirements
+
+  > NOTE: Familiarity with [Kubernetes REST APIs](https://kubernetes.io/docs/reference/) is a prerequisite.
+
+  - Command: `kwctl run -r <"Kubernetes Admission request" file path> -s <"JSON document" file path> <policy URI>`
+
+    - Command breakdown:
+
+      | Required | Option         | Description                                                  | Comment                                                      |
+      | :--------: | -------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+      | ✅        | `-r`           | Path to a file containing the Kubernetes Admission Request to be evaluated | Unfortunately, obtaining Kubernetes Admission Requests requires some extra work. This is a “Kubernetes problem”, we plan to address it in the near future |
+      | -        | `-s`           | Path to a file containing the policy's settings              | It's also possible to use `--settings-json` followed by the settings in JSON format |
+      | ✅        | `<policy URI>` | URI pointing to the policy to be used                        | The supported schemes are: registry://, https://, file://    |
+
+  - Scenario 1:
+
+    - Request to be evaluated: Create a pod with no 'privileged' container
+
+    - Example:
+
+      ```shell
+      $ kwctl run registry://ghcr.io/kubewarden/policies/pod-privileged:v0.1.9 -r unprivileged-pod-request.json
+      {"uid":"C6E115F4-A789-49F8-B0C9-7F84C5961FDE","allowed":true,"status":{"message":""}}
+      ```
+  
+      - Equivalent command with the policy binary downloaded:
+  
+         ```shell
+        `$ kwctl run file://$PWD/pod-privileged-policy.wasm -r unprivileged-pod-request.json
+        {"uid":"C6E115F4-A789-49F8-B0C9-7F84C5961FDE","allowed":true,"status":{"message":""}}
+        ```
+  
+    - Result: The policy allowes the request
+  
+  - Scenario 2:
+  
+    - Request to be evaluated: Create a pod with at least one 'privileged' container
+  
+    - Command: 
+  
+      ```
+      kwctl run registry://ghcr.io/kubewarden/policies/pod-privileged:v0.1.9 -r privileged-pod-request.json
+      ```
+  
+      - Equivalent command with the policy binary downloaded: `kwctl run file://$PWD/pod-privileged-policy.wasm -r privileged-pod-request.json`
+  
+    - Output:
+  
+      ```json
+      {"uid":"8EE6AF8C-C8C8-45B0-9A86-CB52A70EC50D","allowed":false,"status":{"message":"User 'kubernetes-admin' cannot schedule privileged containers"}}
+      ```
+  
+    - Result: The policy denies the request
+  
+    > **INFO**: if you want to see a more complex example, you can read this Kubewarden blog post: [Introducing `kwctl` to Kubernetes Administrators](https://www.kubewarden.io/blog/2021/06/kwctl-intro-for-kubernetes-administrators/)
```

Comment:
```suggestion
    > **NOTE**: If you want to see a more complex example, you can read the Kubewarden blog post [Introducing `kwctl` to Kubernetes Administrators](https://www.kubewarden.io/blog/2021/06/kwctl-intro-for-kubernetes-administrators/).
```

Reply:
It might be a good idea not to have two colons in the same sentence, so just changed the wording a bit to avoid it.